kgo: document KongPluginInstallation (#8034)
Kong/docs.konghq.com: app/_includes/md/custom-plugin.md

Reviewer: Can we link here to a docs page that treats about Kong custom plugins, for example https://docs.konghq.com/gateway/latest/plugin-development/ ?

<!-- Shared between KIC and KGO plugin installation -->
## Create a custom plugin

1. Create a directory with plugin code.

   {:.note}
   > If you already have a real plugin, you can skip this step.

    ```bash
    mkdir myheader
    echo 'local MyHeader = {}

    MyHeader.PRIORITY = 1000
    MyHeader.VERSION = "1.0.0"

    function MyHeader:header_filter(conf)
      -- do custom logic here
      kong.response.set_header("myheader", conf.header_value)
    end

    return MyHeader
    ' > myheader/handler.lua

    echo 'return {
      name = "myheader",
      fields = {
        { config = {
            type = "record",
            fields = {
              { header_value = { type = "string", default = "roar", }, },
            },
        }, },
      }
    }
    ' > myheader/schema.lua
    ```

>    After your plugin code available in a directory, the directory should look like this:

Reviewer: ```suggestion
   After these steps, the directory should look like this:
```
